Automattic/jetpack
# Add Jetpack Backup Daily and Real-time to My Plan page

#### Changes proposed in this Pull Request:
This adds support for the new Jetpack Backup products to the My Plan page.

Changes:
1. Update header title for Jetpack Backup plans.
2. Update Backup card to correctly display Jetpack Backup plans.

Screenshots will be added once I have the final SVG.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
This updates the My Plan page to support the new Jetpack Backup product.

See the P2 here for the design of this PR: p1HpG7-7M8-p2
See the P2 here for the overall MT: p1HpG7-7ET-p2

#### Testing instructions:
The only way to test these products is to fake the data in Redux until the product is available to purchase in the wpcom store (see for status of that dev see p1HpG7-7Dj-p2).

To fake the data use the Dev Tools to choose the "Jetpack Backup Daily" or "Jetpack Backup Real-time" plans.

To test:
1. View the My Plan page for a free plan and some current Jetpack paid plans. Test the upgrade CTA in the header and check that no cards are missing.
2. View the My Plan page for each of the faked Jetpack Backup plans listed above. Test the upgrade CTA in the header, check that the Backup card is displayed and that no cards are missing, and check that the Backup card link works.

#### Proposed changelog entry for your changes:

* Updated My Plan page to support Jetpack Backup Daily and Jetpack Backup Real-time
